requirements.txt

```diff
@@ -0,0 +1 @@
+boto==2.38.0
```

Comment:
Sure, I'm happy to leave the version constraint details up to you. You know better than I do what is and is likely to remain compatible.